tests/e2e/nimbus_utils.go

```diff
@@ -60,6 +61,25 @@ func vMPowerMgmt(user string, location string, podname string, hostList string,
 	return err
 }
 
+/*
+datastoreNimbusOps method is used to perform datatsore nimbus operations
+*/
+func datastoreNimbusOps(user string, location string, podname string, vmName string, op string) error {
```

Comment:
NimbusOps in this method name is not convincing. Can we name it as  operationsOnDatastore Or datatoreOperations